src/assets/CollectiblesController.ts

```diff
@@ -118,6 +118,11 @@ export interface CollectibleMetadata {
   collectionImage?: string;
 }
 
+interface DetectionParams {
+  address: string;
```

Comment:
[nit] - we should distinguish this from the contractAddress for clarity
```suggestion
  userAddress: string;
```

Reply:
Renamed in a80f003